database/seeding.js

```diff
@@ -144,27 +145,34 @@ const database = require('./index.js');
 
     /*--------------------------------*/
     // total one star reviews
-    seedling.aggregateOneStarReview = oneStarReview;
+
+    // MAKE THIS A PERCENTAGE
+    const percentageForOneStar = Math.round((oneStarReview / randomNumberOfReviewsPerProduct) * 100);
```

Comment:
Nice! I created a function in my component that changes the rating to a percentage value. I should do this instead.😎